### ISSUE TITLE
Document other installation options

### ISSUE THREAD

**Issue by magistau** [oven-sh/bun#14073]
### What is the type of issue?

Documentation is missing

### What is the issue?

Suggestions like `curl ... | sh` are usually frowned upon, since running a shell script from the internet is quite a security risk. While it is a possible - and, perhaps, prioritized - installation method, having other options is nice and more trustworthy. As an example, https://rustup.rs/ has an "other installation options" link on the front page. While the front page of Bun does not suggest other options, the documentation has a bit more options, but it only suggests some exotic options, i.e. using npm (user needs npm installed), docker (this is even heavier of a requirement) or homebrew (mostly used on MacOS). [Repology] suggests that nixpkgs (Nix package repository) and AUR (Arch Linux user repository, as opposed to team-maintained) are also available and as the project develops there will be more. As a current Nix user and former Arch user, I think it would help to include these as installation options. Another option is including a repology-provided badge:

[![Packaging status](https://repology.org/badge/vertical-allrepos/bun.svg)](https://repology.org/project/bun/versions)

TL;DR: Add an "other installation options" link on the front page, and include AUR and nixpkgs installation options in docs.

[repology]: https://repology.org/project/bun/versions

### Where did you find it?

- https://bun.sh
- https://bun.sh/docs/installation

**Comment by thodnev** [comment]
Moreover, GNU/Linux distros have their own package managers. Installing software without using a package manager means its dependencies & files will be untracked, which may lead to many problems and frustrations in the future.

It would be nice to have pre-built packages for the most popular GNU/Linux distros.

Nowadays, all popular distros provide very simple ways of packaging the existing software.
You may take a look at the [PKGBUILD script](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=bun-git) for Arch Linux (AUR). That PKGBUILD is heavily outdated btw, limiting the software availability for the users.
